src/frontend/screens/Library/components/InstallModal/SideloadDialog/index.tsx

```diff
@@ -141,7 +141,9 @@ export default function SideloadDialog({
       art_cover: imageUrl,
       is_installed: true,
       art_square: imageUrl,
-      canRunOffline: true
+      canRunOffline: true,
+      is_linux_native: gameInfo.is_linux_native ?? false,
+      is_mac_native: gameInfo.is_mac_native ?? false
```

Comment:
Any different and better approach for this? 
Will defaulting it to false cause any issues? I didn't encounter any.